src/components/content/Content.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import Hello from '../hello';
+import Hello from './hello';
 import Dashboard from '../Dashboard';
 
 import '../content/content.scss';
```

Comment:
why do you need to exit from current folder and then again dive into it? make just `import './contents.scss'`